Make.inc

```diff
@@ -1499,8 +1499,12 @@ LIBGCC_NAME := libgcc_s_seh-1.$(SHLIB_EXT)
 endif
 endif
 ifeq ($(OS),Darwin)
+ifeq ($(ARCH),aarch64)
+LIBGCC_NAME := libgcc_s.2.$(SHLIB_EXT)
```

Comment:
It's not clear to me whether this `libgcc` name is a temporary thing from iain's branch or permanent.

Reply:
Not clear to me either, but since we're hardcoding the toolchain for now, I think it's ok.